### PR TITLE
[IMP] core: additionnal country db for geoip

### PR DESCRIPTION
**Pull request by Julien00859**
Task: 2848206

### CODE REVIEW

**Comment by Julien00859** [comment]
We found a way to include the doc commit in the batch :)

File: content/developer/misc/other/cmdline.rst [hunk 1]
```diff
@@ -129,9 +129,13 @@ Running the server
...
 
-.. option:: --geoip-db <path>
+.. option:: --geoip-city-db <path>
 
-   Absolute path to the GeoIP database file.
+   Absolute path to the city GeoIP database file.
+
+.. option:: --geoip-country-db <path>
+
+   Absolute path to the country GeoIP database file.
```
Comment (AntoineVDV) [inline content/developer/misc/other/cmdline.rst:138]: As per maxmind.com nomenclature.
```suggestion
   Absolute path to the GeoIP City database file.

.. option:: --geoip-country-db <path>

   Absolute path to the GeoIP Country database file.
```

**Review by AntoineVDV — COMMENTED** [review]
Content review for `content/developer/misc/other/cmdline.rst` only. @odoo/website-doc-review please reset my request for review when the PR is ready for technical review ;)

**Review by Lau-Odoo — CHANGES_REQUESTED** [review]
- Please, adapt the number of "=" symbols to your title length.
- Only one commit per PR please :)

**Review by Lau-Odoo — APPROVED** [review]

File: content/applications/websites/website/publish/on-premise_geo-ip-installation.rst [hunk 2]
```diff
@@ -5,60 +5,55 @@ Geo IP Installation (On-Premises Database)
 Installation
 ============
 
-.. warning::
-   Please note that the installation depends on your computer's operating system and distribution.
-   We will assume that a Linux operating system is being used.
+1. Download both the GeoLite2 City and Country
+   `databases <https://dev.maxmind.com/geoip/geoip2/geolite2/>`_. You should end up with two files
+   called ``GeoLite2-City.mmdb`` and ``GeoLite2-Country.mmdb``
```
Comment (AntoineVDV) [inline content/applications/websites/website/publish/on-premise_geo-ip-installation.rst:10]: We have a task to highlight file names differently so let's start flagging them with the `file` role already :)
```suggestion
   called :file:`GeoLite2-City.mmdb` and file:`GeoLite2-Country.mmdb`
```

File: content/applications/websites/website/publish/on-premise_geo-ip-installation.rst [hunk 3]
```diff
@@ -5,60 +5,55 @@ Geo IP Installation (On-Premises Database)
...
 
-#. Install `geoip2 <https://pypi.org/project/geoip2/>`__ Python library
-    .. code-block:: bash
-
-      pip install geoip2
-
-#. Download the `GeoLite2 City database <https://dev.maxmind.com/geoip/geoip2/geolite2/>`_. You
-   should end up with a file called ``GeoLite2-City.mmdb``
-#. Move the file to the folder ``/usr/share/GeoIP/``
+2. Move the files to the folder ``/usr/share/GeoIP/``
```
Comment (AntoineVDV) [inline content/applications/websites/website/publish/on-premise_geo-ip-installation.rst:12]:
```suggestion
2. Move the files to the folder :file:`/usr/share/GeoIP/`
```

File: content/applications/websites/website/publish/on-premise_geo-ip-installation.rst [hunk 4]
```diff
@@ -5,60 +5,55 @@ Geo IP Installation (On-Premises Database)
...
+        mv ~/Downloads/GeoLite2-Country.mmdb /usr/share/GeoIP/
 
-#. Restart the server
+3. Restart the server
 
 .. note::
-   If you can't/don't want to locate the geoip database in ``/usr/share/GeoIP/``, you can use the
-   ``--geoip-db`` option of the Odoo command line interface. This option takes the absolute path to
-   the GeoIP database file and uses it as the GeoIP database. For example:
+   If you don't want to locate the geoip database in ``/usr/share/GeoIP/``, use the
```
Comment (AntoineVDV) [inline content/applications/websites/website/publish/on-premise_geo-ip-installation.rst:21]:
```suggestion
   If you don't want to locate the geoip database in :file:`/usr/share/GeoIP/`, use the
```

File: content/applications/websites/website/publish/on-premise_geo-ip-installation.rst (hunk 3 above)
Comment (AntoineVDV) [inline content/applications/websites/website/publish/on-premise_geo-ip-installation.rst:12]:
```suggestion
2. Move the files to the folder ``/usr/share/GeoIP/``.
```

File: content/applications/websites/website/publish/on-premise_geo-ip-installation.rst [hunk 5]
```diff
@@ -5,60 +5,55 @@ Geo IP Installation (On-Premises Database)
...
-      pip install geoip2
-
-#. Download the `GeoLite2 City database <https://dev.maxmind.com/geoip/geoip2/geolite2/>`_. You
-   should end up with a file called ``GeoLite2-City.mmdb``
-#. Move the file to the folder ``/usr/share/GeoIP/``
+2. Move the files to the folder ``/usr/share/GeoIP/``
     .. code-block:: bash
 
         mv ~/Downloads/GeoLite2-City.mmdb /usr/share/GeoIP/
+        mv ~/Downloads/GeoLite2-Country.mmdb /usr/share/GeoIP/
```
Comment (AntoineVDV) [inline content/applications/websites/website/publish/on-premise_geo-ip-installation.rst:16]: While you're at it, remove the extra indentation space before each of these lines that makes item 2. behave as a quote block. You'll also need to add an extra blank line after item 2.
```suggestion

   .. code-block:: bash

       mv ~/Downloads/GeoLite2-City.mmdb /usr/share/GeoIP/
       mv ~/Downloads/GeoLite2-Country.mmdb /usr/share/GeoIP/
```

File: content/applications/websites/website/publish/on-premise_geo-ip-installation.rst [hunk 6]
```diff
@@ -5,60 +5,55 @@ Geo IP Installation (On-Premises Database)
...
 2. Choose :menuselection:`Customize --> HTML/CSS/JS Editor`.
 3. Add the following piece of XML in the page :
 
 .. code-block:: xml
 
-    <h1 class="text-center" t-esc="request.session.get('geoip')"/>
+    <h1 class="text-center" t-esc="request.geoip.country.name or 'geoip failure'"/>
 
-You should end up with a dictionary indicating the location of the IP address.
+4. Save and refresh the page.
```
Comment (AntoineVDV) [inline content/applications/websites/website/publish/on-premise_geo-ip-installation.rst:48]:
```suggestion
#. Save and refresh the page.
```
You'll most like need to properly indent the code block above to correctly align it with item 3. as it's currently misaligned.

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

File: content/applications/websites/website/publish/on-premise_geo-ip-installation.rst [hunk 7]
```diff
@@ -5,60 +5,56 @@ Geo IP Installation (On-Premises Database)
...
-      pip install geoip2
+#. Move the files to the folder :file:`/usr/share/GeoIP/`.
 
-#. Download the `GeoLite2 City database <https://dev.maxmind.com/geoip/geoip2/geolite2/>`_. You
-   should end up with a file called ``GeoLite2-City.mmdb``
-#. Move the file to the folder ``/usr/share/GeoIP/``
     .. code-block:: bash
 
         mv ~/Downloads/GeoLite2-City.mmdb /usr/share/GeoIP/
+        mv ~/Downloads/GeoLite2-Country.mmdb /usr/share/GeoIP/
```
Comment (AntoineVDV) [inline content/applications/websites/website/publish/on-premise_geo-ip-installation.rst:17]: See https://github.com/odoo/documentation/pull/2151#discussion_r896691507
image
```suggestion
   .. code-block:: bash

       mv ~/Downloads/GeoLite2-City.mmdb /usr/share/GeoIP/
       mv ~/Downloads/GeoLite2-Country.mmdb /usr/share/GeoIP/
```

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Comment by AntoineVDV** [comment]
@Julien00859 anything worth checking?

**Comment by Julien00859** [comment]
Nothing changed, it was just a rebase, thank for asking :)

**Comment by Julien00859** [comment]
@robodoo r+

**Pull request closed without merging**